lib/tpm2_identity_util.c

```diff
@@ -0,0 +1,480 @@
+//**********************************************************************;
+// Copyright (c) 2017, Intel Corporation
```

Comment:
Should't this be MIT? Or did you take this code from Intel?

Reply:
I see now that you took some bits from `tpm2_import.c`, I should had go through all the patch before commenting. In that case I think you should add both Copyrights since you are also adding new code.